projects/br_scs/test/Mirza/Common/Tests/InitClient.hs

```diff
@@ -140,27 +139,21 @@ newBusinessToBusinessResponse =
 
 
 newUserToBasicAuthData :: BT.NewUser -> BasicAuthData
-newUserToBasicAuthData newUser = BasicAuthData (toByteString $ BT.newUserEmailAddress newUser) ""
+newUserToBasicAuthData _newUser = BasicAuthData "" "" -- TODO: Extract info from or associated with newUser.
```

Comment:
I would suggest completely removing this function and dealing with the ~consequences~ compile errors, since we are not using `BasicAuthData` anymore.

Reply:
Same as https://github.com/data61/Mirza/pull/430#discussion_r281875409 . Basically yep thats the plan I just haven't got to that stage yet.